netbox_plugin_extensions/tables/columns.py

```diff
@@ -11,14 +11,19 @@ class PluginButtonsColumn(ButtonsColumn):
     """
 
     template_code = """
-    {{% load plugin_buttons %}}
```

Comment:
I feel like this is a step back as these button helpers are used in other code.

What is the reasoning behind going this route?